src/dappsExplorer/NoResults.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { DappExplorerEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Touchable from 'src/components/Touchable'
+import { DappFilter } from 'src/dapps/types'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  filter: DappFilter | null
+  removeFilter: () => void
+  testID: string
+}
+
+export function NoResults({ filter, removeFilter, testID }: Props) {
+  const { t } = useTranslation()
+
+  const removePress = () => {
+    ValoraAnalytics.track(DappExplorerEvents.dapp_filter_remove)
+    removeFilter()
+  }
+
+  return (
+    <View testID={`${testID}/NoResults`} style={styles.favorites}>
+      <Text style={styles.filterAppliedText}>
+        <Trans
+          i18nKey={'dappsScreen.emptyResults.message'}
+          tOptions={{ filter: filter?.name.toUpperCase() }}
+        >
+          <Text style={styles.filterText} />
+        </Trans>
+      </Text>
+      <Touchable
+        onPress={removePress}
+        style={styles.removeFilterTouchable}
+        testID={`${testID}/NoResults/RemoveFilter`}
+      >
+        <Text style={styles.removeFilterText}>{t('dappsScreen.emptyResults.removeFilter')}</Text>
+      </Touchable>
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  favorites: {
```

Comment:
`container`?

Reply:
Fixed in e5341bcc2662f046e048379299861b43e3205765!